pkg/cmd/kind/get/clusters/clusters.go

```diff
@@ -50,8 +50,12 @@ func runE(logger log.Logger, streams cmd.IOStreams) error {
 	if err != nil {
 		return err
 	}
-	for _, cluster := range clusters {
-		fmt.Fprintln(streams.Out, cluster)
+	if len(clusters) > 0 {
+		for _, cluster := range clusters {
+			fmt.Fprintln(streams.Out, cluster)
+		}
+	} else {
+		fmt.Fprintln(streams.Out, "No clusters available")
```

Comment:
This should not go to stdout. It's a user facing message, not program output.
We could use Logger.Warn maybe.

Reply:
Sure, I just changed to `logger.Warn` in latest push.